### PR TITLE
follow ups reminder count

### PR DESCRIPTION
**Pull request by NirmalNegi07**
#2124

### Description
Follow-Ups reminder count.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by P4NK4J — CHANGES_REQUESTED** [review]
@NirmalNegi07 based on our conversation earlier, please make the changes and assign for review once done.

File: Modules/HR/Console/Recruitment/SendFollowUpThresholdMail.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    {
+        $applications = Application::whereIn('status', ['new', 'in-progress'])->get();
+        $applications = $applications->reject(function ($application) {
+            $applicationround = $application->latestApplicationRound;
+            if ($applicationround) {
+                $followUpCount = $application->latestApplicationRound->followUps->count();
+                if ($followUpCount == config('hr.follow-up-attempts-threshold')) {
+                    return $application;
+                }
+            }
```
Comment (P4NK4J) [inline Modules/HR/Console/Recruitment/SendFollowUpThresholdMail.php:52]: How about ?
```suggestion
            $followUpCount = $application->latestApplicationRound ? $application->latestApplicationRound->followUps->count() : 0;
            if ($followUpCount == config('hr.follow-up-attempts-threshold')) {
                return $application;
            }
```

File: Modules/HR/Emails/SendThreshholdFollowUp.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this->from(config('hr.default.email'), config('hr.default.name'))
```
Comment (P4NK4J) [inline Modules/HR/Emails/SendThreshholdFollowUp.php:33]: Why two email addresses in sender's address ?

File: Modules/HR/Resources/views/application/followups.blade.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div>
+	<p>Hey {{$user->name}},
+	<p>Please check with the following applicants, we need to follow up with them via phone call. They have already been sent follow-ups via email. If possible please check with them over a phone call regarding their interview schedules.
+    </p>
```
Comment (P4NK4J) [inline Modules/HR/Resources/views/application/followups.blade.php:4]: Why 2 `p` tags are opening up and only 1 is closed ? Please see the indentation as well

File: app/Console/Kernel.php [hunk 4]
```diff
@@ -38,6 +38,7 @@ protected function schedule(Schedule $schedule)
         $schedule->command('sync:effortsheet')->weekdays()->timezone(config('constants.timezone.indian'))->everyFourHours();
         $schedule->command('effort-summary:send')->weekdays()->timezone(config('constants.timezone.indian'))->at('21:00');
         $schedule->command('hr:check-follow-ups')->daily();
+        $schedule->command('hr:send-follow-ups-mail')->daily();
```
Comment (P4NK4J) [inline app/Console/Kernel.php:41]: Can we have a fixed time like in the morning 8:00 am ?

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left some comments, please check

File: app/Console/Kernel.php (hunk 4 above)
Comment (NirmalNegi07) [reply app/Console/Kernel.php:41]: yes sir i will do it.

File: Modules/HR/Resources/views/application/followups.blade.php (hunk 3 above)
Comment (NirmalNegi07) [reply Modules/HR/Resources/views/application/followups.blade.php:4]: I will fix it.

File: Modules/HR/Console/Recruitment/SendFollowUpThresholdMail.php (hunk 1 above)
Comment (NirmalNegi07) [reply Modules/HR/Console/Recruitment/SendFollowUpThresholdMail.php:52]: Looks nice

File: Modules/HR/Emails/SendThreshholdFollowUp.php (hunk 2 above)
Comment (NirmalNegi07) [reply Modules/HR/Emails/SendThreshholdFollowUp.php:33]: Sir, one is email address and another one is name.

File: Modules/HR/Config/config.php [hunk 5]
```diff
@@ -170,4 +170,9 @@
         'start_date' => '2022-07-06'
     ],
     'non-verified-application-start-date' => '2022-07-06',
+    'follow-up-attempts-threshold' => '2',
+    'hr-followup-email' => [
+        'primary' => env('HR_FOLLOWUP_EMAIL_PRIMARY', 'deepak.sharma@coloredcow.in'),
+        'secondary' => env('HR_FOLLOWUP_EMAIL_SECONDARY', 'pk@coloredcow.in'),
```
Comment (P4NK4J) [inline Modules/HR/Config/config.php:176]:
```suggestion
        'secondary' => env('HR_FOLLOWUP_EMAIL_SECONDARY', 'pk@coloredcow.com'),
```

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Comment by P4NK4J** [comment]
@NirmalNegi07 Glad you updated the env example file along with your changes. Good job with the overall work 👏

**Comment by P4NK4J** [comment]
@NirmalNegi07 Glad you updated the env example file along with your changes. Good job with the overall work 👏

**Pull request merged**